### PR TITLE
feat(s2s): address PR review comments from @ronaldmansveld

### PR DESCRIPTION
**Pull request by Daniel-Lam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-1970

### Change description ###
Address PR review comments from @ronaldmansveld that didn't make the cut on PR #17.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/auth/s2s/s2s.class.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        microservice: 'rpx-xui',
+        s2sEndpointUrl: 'http://s2s.local',
+        s2sSecret: 'topSecret',
+    } as S2SConfig
+    // const token = 'abc123'
+
+    beforeEach(() => {
+        s2sAuth = new S2SAuth()
+        s2sAuth.configure(s2sConfig)
+        // sinon.stub(serviceAuth, 'postS2SLease').resolves(token)
```
Comment (paddybasi) [inline src/auth/s2s/s2s.class.spec.ts:29]: know we know how to properly mock any API calls, can we please re-enable the tests and implement?
E.g. here
```
jest.spyOn(serviceAuth, 'postS2SLease').mockImplementation(() => Promise.resolve(token))
```

File: tsconfig.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "declaration": true,
     "outDir": "./dist",
     "strict": true,
-    "typeRoots": ["node_modules/@types"]
+    "typeRoots": ["node_modules/@types", "customTypes"]
```
Comment (paddybasi) [inline tsconfig.json:12]: Is this still required?

Comment (Daniel-Lam) [reply tsconfig.json]: @paddybasi - oh no, I forgot that too! :)

File: src/auth/s2s/s2s.class.spec.ts (hunk 1 above)
Comment (Daniel-Lam) [reply src/auth/s2s/s2s.class.spec.ts]: Sure - will work on this as part of a new PR for the ticket covering remaining unit test activities.

**Pull request merged**